sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/convert/MappingCosmosConverter.java

```diff
@@ -96,6 +99,27 @@ public void write(Object source, JsonNode sink) {
         }
     }
 
+    private <R> R readJsonNodeAsValue(JsonNode jsonNode, Class<R> type) {
```

Comment:
We can get rid of this code and just use objectMapper.treeToValue

Reply:
Done.